Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3-onbuild
+
+COPY ./pgoapi /pgoapi
+COPY ./pokecli.py /pokecli.py
+
+RUN pip install requests
```

Comment:
I totally missed this file - I'm not a Pythoner :)
The pip install over each line should do the trick I guess. See commit below.